app/notification_test.go

```diff
@@ -1144,56 +1126,56 @@ func TestGetMentionKeywords(t *testing.T) {
+	ids, ok = mentions["First"]
+	require.False(t, !ok || len(ids) != 2 || (ids[0] != user2.Id && ids[1] != user2.Id) || (ids[0] != user4.Id && ids[1] != user4.Id),
+		"should've mentioned user2 and user4 with First")
+	ids, ok = mentions["@channel"]
+	require.False(t, !ok || len(ids) != 2 || (ids[0] != user3.Id && ids[1] != user3.Id) || (ids[0] != user4.Id && ids[1] != user4.Id),
+		"should've mentioned user3 and user4 with @channel")
+	ids, ok = mentions["@all"]
+	require.False(t, !ok || len(ids) != 2 || (ids[0] != user3.Id && ids[1] != user3.Id) || (ids[0] != user4.Id && ids[1] != user4.Id),
+		"should've mentioned user3 and user4 with @all")
 
 	// multiple users and more than MaxNotificationsPerChannel
 	mentions = th.App.getMentionKeywordsInChannel(profiles, false, channelMemberNotifyPropsMap4Off)
-	if len(mentions) != 4 {
-		t.Fatal("should've returned four mention keywords", mentions)
-	} else if _, ok := mentions["@channel"]; ok {
-		t.Fatal("should not have mentioned any user with @channel")
-	} else if _, ok := mentions["@all"]; ok {
-		t.Fatal("should not have mentioned any user with @all")
-	} else if _, ok := mentions["@here"]; ok {
-		t.Fatal("should not have mentioned any user with @here")
-	}
-
+	require.False(t, len(mentions) != 4, "should've returned four mention keywords")
+	_, ok = mentions["@channel"]
+	require.False(t, ok, "should not have mentioned any user with @channel")
+	_, ok = mentions["@all"]
+	require.False(t, ok, "should not have mentioned any user with @all")
+	_, ok = mentions["@here"]
+	require.False(t, ok, "should not have mentioned any user with @here")
 	// no special mentions
 	profiles = map[string]*model.User{
 		user1.Id: user1,
 	}
 	mentions = th.App.getMentionKeywordsInChannel(profiles, false, channelMemberNotifyPropsMap4Off)
-	if len(mentions) != 3 {
-		t.Fatal("should've returned three mention keywords")
-	} else if ids, ok := mentions["user"]; !ok || len(ids) != 1 || ids[0] != user1.Id {
-		t.Fatal("should've mentioned user1 with user")
-	} else if ids, ok := mentions["@user"]; !ok || len(ids) != 2 || ids[0] != user1.Id || ids[1] != user1.Id {
-		t.Fatal("should've mentioned user1 twice with @user")
-	} else if ids, ok := mentions["mention"]; !ok || len(ids) != 1 || ids[0] != user1.Id {
-		t.Fatal("should've mentioned user1 with mention")
-	} else if _, ok := mentions["First"]; ok {
-		t.Fatal("should not have mentioned user1 with First")
-	} else if _, ok := mentions["@channel"]; ok {
-		t.Fatal("should not have mentioned any user with @channel")
-	} else if _, ok := mentions["@all"]; ok {
-		t.Fatal("should not have mentioned any user with @all")
-	} else if _, ok := mentions["@here"]; ok {
-		t.Fatal("should not have mentioned any user with @here")
-	}
+	require.False(t, len(mentions) != 3, "should've returned three mention keywords")
+	ids, ok = mentions["user"]
+	require.False(t, !ok || len(ids) != 1 || ids[0] != user1.Id, "should've mentioned user1 with user")
```

Comment:
Again, here would be a good idea to look at breaking down these statements into multiple.